docker/dockerfile.tf

```
@@ -15,7 +15,7 @@ FROM ${BASE_IMAGE} as base
 COPY --chown=1000:1000 --from=triton /opt/tritonserver/backends/tensorflow2 backends/tensorflow2/
 
 # Tensorflow dependencies (only)
-RUN pip install tensorflow-gpu \
+RUN pip install tensorflow-gpu  scikit-build \
```

You shouldn't have to install scikit-build yourself

```suggestion
RUN pip install tensorflow-gpu  scikit-build \
```
```suggestion
RUN pip install tensorflow-gpu  \
```